lib/puppet/version.rb

```diff
@@ -6,7 +6,7 @@
 # Raketasks and such to set the version based on the output of `git describe`
 
 module Puppet
-  PUPPETVERSION = '7.32.0'
+  PUPPETVERSION = '7.32.1'
```

Comment:
I think we want to keep version `7.33.0` here and below in preparation for the next release.